src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Sql/Sprocs/ResetTask_2.sql

```diff
@@ -33,14 +33,12 @@ SELECT @retryCount = RetryCount,
 FROM   [dbo].[TaskInfo]
 WHERE  TaskId = @taskId
        AND RunId = @runId;
```

Comment:
Though SELECT and UPDATE are in the same transaction it is not guaranteed that values did not change. Please move all logic to single UPDATE.

Reply:
Fixed